src/olympia/addons/tests/test_admin.py

```diff
@@ -1118,3 +1124,101 @@ def test_can_delete(self):
             f"[{restriction.addon.id}] deleted: ['FR']"
         )
         assert mail.outbox[0].to == ['amo-admins@mozilla.com']
+
+
+class TestAddonBrowserMappingAdmin(TestCase):
+    def setUp(self):
+        user = user_factory(email='someone@mozilla.com')
+        self.grant_permission(user, '*:*')
```

Comment:
We should start with a more specific permission. I suggest re-using `ADMIN_CURATION` (need to edit `DJANGO_PERMISSIONS_MAPPING`).